src/ms-review.tex

```diff
@@ -367,6 +367,7 @@ \section{Introduction}
 \cite[see e.g.][]{Mohrmann2019}. The \hawc collaboration also released
 a limited test dataset of the Crab Nebula, which was used to validate
 the \gammapy package in \cite{Olivera2022}.
+The increased availability of public data that followed the definition of a common data format, and the development of \gammapy as a community-driven open software, led the way toward a more open science in the very-high-energy \gray Astronomy domain. In future CTAO will be an open observatory committed to follow the FAIR (Findable, Accessible, Interoperable and Reusable) principles \citep{FAIR16, FAIR22} that define the key requirements for open science. 
```

Comment:
- related comment:
I am wondering if we should also mention already here that this necessity does not only come from the fact of being ‘open observatories’, but also from open observatories following and implementing the FAIR principles.

As part of this, I am missing some statement on GammaPy in relation to FAIR principles, e.g. the FAIR4RS. At least a link should exist from somewhere / mentioned in the context:

https://www.nature.com/articles/s41597-022-01710-x

Reply:
For the FAIR4RS, here a nice reference: https://doi.org/10.15497/RDA00068
This is still an objective, because no fair evaluation has been made about our compliance..